docs/concepts/05_credentials/05_verification.md

```diff
@@ -3,68 +3,111 @@ id: verification
 title: Verification
 ---
 
-KILT allows a Verifier to check if the information in a credential presented by a Claimer is correct and valid.
+KILT lets a Verifier check if the information in a credential presented by a Claimer is correct and valid.
 With the presentation of the credential, the Claimer also presents evidence that a third party (i.e., an Attester) ensured the correctness of the Claimer’s attributes.
 
-The Verifier trusts this third party either because they trust their reputation directly or they trust a delegation structure that this Attester is part of (e.g., a State department issuing driving licenses).
+The Verifier trusts this third party either because they trust their reputation directly or they trust a delegation structure that this Attester is part of.
+For example, a State department issuing driving licenses.
 
 For the verification process:
 
-- The Claimer needs their credential and the private key associated with their identifier
-- The Verifier needs the identifier of the trusted Attester
+-   The Claimer needs their credential and the private key associated with their identifier
+-   The Verifier needs the identifier of the trusted Attester
 
-During the verification process the Claimer wants to prove three things to the Verifier:
+During the verification process the Claimer wants to prove the following things to the Verifier:
 
-- The credential is valid (i.e., not revoked by its Attester)
-- The attributes in the credential actually refer to the Claimer 
-- The credential contains information that is relevant for the Verifier in this use case
+-   The credential is valid (i.e., not revoked by its Attester)
+-   The attributes in the credential actually refer to it's Presenter
+-   The credential contains information relevant for the Verifier for this use case
+- That an Attester ensured the correctness of the Claimer's attributes
```

Comment:
Maybe add _trustworthy_ to it? 

Not so important though.

Reply:
```suggestion
- That an Attester ensured the correct and trustworthy-ness of the Claimer's attributes
```